src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/xunitrunner-perf.py

```diff
@@ -33,10 +43,15 @@ def _write_output_path(file_path, settings):
         shutil.copy2(file_path, output_path)
         return output_path
     else:
-        fc = helix.azure_storage.get_upload_client(settings)
-        return fc.upload(file_path, os.path.basename(file_path))
-
-def _prepare_execution_environment(settings, framework_in_tpa):
+        try:
+            fc = helix.azure_storage.get_upload_client(settings)
+            url = fc.upload(file_path, os.path.basename(file_path))
+            return url
+        except ValueError, e:
+            event_client = helix.event.create_from_uri(settings.event_uri)
+            event_client.error(settings, "FailedUpload", "Failed to upload "+file_path+"after retry", None)
```

Comment:
These are directly copied lines from latest changes to xunitrunner, will make the changes.